src/composite/MegaChart.js

```diff
@@ -88,6 +88,11 @@
         return this;
     };
 
+    MegaChart.prototype.downloadCSV = function () {
+        Utility.downloadBlob("CSV", this._chart.export("CSV"));
```

Comment:
does downloadBlob handle user response if for some reason the data stream isn't available?